roles/lib_openshift/src/doc/sdnvalidator

```diff
@@ -0,0 +1,27 @@
+# flake8: noqa
+# pylint: skip-file
```

Comment:
Sorry, I assumed the files were being concatenated as-is, without having looked at the code.
I see now that these lines are filtered out when generating the files: https://github.com/rhcarvalho/openshift-ansible/blob/master/roles/lib_openshift/src/generate.py#L72